com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1121,6 +1121,8 @@ protected override void Render(ScriptableRenderContext renderContext, Camera[] c
 #endif
             Terrain.GetActiveTerrains(m_ActiveTerrains);
 
+            ProbeReferenceVolume.instance.Update();
```

Comment:
We already have two sort of updates already. 

We have the pending operation stuff (called every frame) and we have a SortPendingCells in the Lightloop. We should integrate the streaming in one of the two. I am fine with having the `SortPendingCells ` into the new `Update` but also I think it should live in the lightloop as it is lighting related only.

Reply:
Didn't really think too hard on this one so I can change that.
The only constraint we have is that it needs to be called once for all cameras at some point because we'll need to factor N cameras to derive the proper streaming needs for a frame.

Reply:
So I took a closer look and both SortPendingCells and the pending loading stuff are called per camera. So I can't use that for stream. I don't see any other way then.

Reply:
Currently is done per camera because it is a quick hack, it should follow the same heuristic of (what's the highest priority cell) that streaming is using. 

The sorting happens on the monolithic system still, so 2 camera stomp each other (it is a rough and very temp system before streaming has a good heuristic)

Reply:
Hm all right. Ok then, I'm removing this one and I'll probably have to change the existing in my next PRs then.